### PR TITLE
Ccs 5059 adding local preview tags

### PR DESCRIPTION
**Pull request by rlahoda**
This adds in some comments to the `document.html.haml` file to allow the output code to be modified for local preview with Juno

### CODE REVIEW

**Comment by wesruv** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr15-npeoh8dpluaxzgvjkho4grz68empv2zw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61e7007df67b7c778c119641

File: templates/document.html.haml [hunk 1]
```diff
@@ -50,7 +51,6 @@
       - if attr? key
         %meta{:name=>(meta || key), :content=>(attr key)}
 
-    %title=((doctitle :sanitize => true) || (attr 'untitled-label'))
```
Comment (wesruv) [inline templates/document.html.haml:53]: Did you mean to remove the `<title>` tag?

**Review by wesruv — CHANGES_REQUESTED** [review]

File: templates/document.html.haml (hunk 1 above)
Comment (rlahoda) [reply templates/document.html.haml:53]: there's a title tag getting added by cp-chrome

**Comment by wesruv** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr15-npeoh8dpluaxzgvjkho4grz68empv2zw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61e7007df67b7c778c119641

**Comment by wesruv** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr15-npeoh8dpluaxzgvjkho4grz68empv2zw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61e7007df67b7c778c119641

File: examples/assembly_access-control-list.html [hunk 2]
```diff
@@ -184,9 +188,11 @@ <h2 id="_additional_resources">Additional resources</h2>
 <a class="rh-docs-legal-notice__link" href="https://access.redhat.com/docs-legal-notice">Legal Notices for Trademarks</a>
 </div>
 </article>
+<!-- inject:footer:html -->
```
Comment (wesruv) [inline examples/assembly_access-control-list.html:192]: Looks like the footer is inside of the cp-documentation tags, we should pull that out, into the doc

**Comment by wesruv** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr15-npeoh8dpluaxzgvjkho4grz68empv2zw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61e7007df67b7c778c119641

**Comment by wesruv** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr15-npeoh8dpluaxzgvjkho4grz68empv2zw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61e7007df67b7c778c119641

**Comment by wesruv** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr15-npeoh8dpluaxzgvjkho4grz68empv2zw.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61e7007df67b7c778c119641

**Review by wesruv — APPROVED** [review]

**Pull request merged**